package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "sonarplanet-services",
+  "version": "1.0.0",
+  "description": "Address tracker",
+  "main": "app.js",
+  "repository": "git@github.com:spacelephant/sonarplanet-services.git",
+  "author": "spacelephant",
+  "license": "ISC",
```

Comment:
`Apache-2.0` instead
https://spdx.org/licenses/Apache-2.0.html for the license file